docs/ArbOS_Formats.md

```diff
@@ -107,6 +107,16 @@ Type-specific data:
 
 This message type is reserved for internal use by ArbOS. It should never appear in the inbox.
 
+##### Message type 7: Eth deposit transaction
+
+This message type first transfers the given amount of Eth to the sender's account. Then it executes the given transaction.
+
+Type-specific data:
+
+- Unsigned Transaction or Contract transaction as defined in the L2 message section below
+
+This message type is reserved for internal use by ArbOS. It should never appear in the inbox.
```

Comment:
This one should sometimes appear in the inbox, right?

Reply:
Whoops that was a copy/paste error. I removed that line